src/Initializer/CellLocalMatrices.cpp

```diff
@@ -164,6 +166,36 @@ void seissol::initializers::initializeCellLocalMatrices( MeshReader const&
         MeshTools::normalize(tangent1, tangent1);
         MeshTools::normalize(tangent2, tangent2);
 
+        real NLocalData[6*6];
+        seissol::model::getBondMatrix(normal, tangent1, tangent2, NLocalData);
+        switch (material[cell].local.getMaterialType()) {
+          case seissol::model::MaterialType::anisotropic: {
+            seissol::model::getTransposedGodunovState(  seissol::model::getRotatedMaterialCoefficients(NLocalData, *dynamic_cast<seissol::model::AnisotropicMaterial*>(&material[cell].local)),
+                                                        seissol::model::getRotatedMaterialCoefficients(NLocalData, *dynamic_cast<seissol::model::AnisotropicMaterial*>(&material[cell].neighbor[side])),
+                                                        cellInformation[cell].faceTypes[side],
+                                                        QgodLocal,
+                                                        QgodNeighbor );
+            seissol::model::getTransposedCoefficientMatrix( seissol::model::getRotatedMaterialCoefficients(NLocalData, *dynamic_cast<seissol::model::AnisotropicMaterial*>(&material[cell].local)), 0, AT_tilde );
+            break;
+          }
+          case seissol::model::MaterialType::elastic:
+          case seissol::model::MaterialType::elastoplastic: 
+          case seissol::model::MaterialType::viscoelastic:
+          case seissol::model::MaterialType::viscoplastic: {
```

Comment:
Make clear that fallthrough is intended, this is marked by some analysis tools as potential bug
(in c++17 we can use [[fallthrough]])